storer/s3helpers.go

```diff
@@ -0,0 +1,83 @@
+package storer
```

Comment:
I would call it s3handler and not s3helper. It's more common in the code examples that I saw.